lib/msal-node/src/cache/Storage.ts

```diff
@@ -49,80 +39,118 @@ export class Storage extends CacheManager {
         this.changeEmitters.forEach(func => func.call(null));
     }
 
+    /**
+     * Converts cacheKVStore to InMemoryCache
+     * @param cache
+     */
+    cacheToInMemoryCache(cache: CacheKVStore): InMemoryCache {
+
+        let inMemoryCache: InMemoryCache = {
+            accounts: {},
+            idTokens: {},
+            accessTokens: {},
+            refreshTokens: {},
+            appMetadata: {},
+        };
+
+        for (let key in cache) {
```

Comment:
Should this be a switch statement?

Reply:
I don't think we can equate a value: `cache[key` which will be the `switch(value)` to `instanceof <some object>`. Hence the choice of `if`.